libs/features/icons/icons.css

```diff
@@ -107,6 +107,13 @@
   display: block;
 }
 
+@supports (-webkit-backdrop-filter: blur(1px)) and (not (backdrop-filter: blur(1px))) {
+  .milo-tooltip:focus::before {
```

Comment:
This is the experience I got when trying this out, with an odd transparent space around the element, followed by what I assume is an outline? Would this be the expected outcome?

<img width="432" height="124" alt="Image" src="https://github.com/user-attachments/assets/05784657-0ca9-4db3-8628-4162db1f85a3" />